crates/testing/src/spinning_task.rs

```diff
@@ -148,6 +152,9 @@ where
                                         )
                                         .await
                                     }
+                                    LateNodeContext::Restart => {
+                                        panic!("Cannot spin up a node with Restart context")
```

Comment:
Just for my own understanding: late start is from genesis, right? So a node would delay startup for some period of time then catch up? Is that why restart wouldn't make sense for that node?

Reply:
Yeah late start is a node that never is turned on, restart is a node that is stopped then restarted with only info from `storage`

Reply:
Also here we have to panic because we just don't have the info to create another node